File: conditionals.js
/* 

if (condition) {
    sentences
}

if (condition) {
    sentences
} else {
    sentences
}

if (condition) {
    sentences
} else if (condition) {
    sentences
} else {
    sentences
}

and => &&
or => ||
not => !
 
*/
let a = 5;
let b = 10;
let c = 6;

if (a === b) {
    console.log(a === b)
}

if (a > b) {
    console.log("Verdadero")
} else {
    console.log("Falso")
}

if (a > b && a > c){
    console.log("A es mayor")
} else if (b > c){
    console.log("B es mayor")
} else {
    console.log("C es mayor")
}

File: conditionals.py
'''

if condition:
    sentences

if condition:
    sentences
else: 
    sentences

if condition:
    sentences
elif condition:
    sentences
else: 
    sentences

&& => and
|| => or
! => not

'''
a = 5
b = 10
c = 6

if a == b:
    print(a==b)


if a > b:
    print("Verdadero")
else:
    print("Falso")


if a > b and a > c:
    print("A es mayor")
elif b > c:
    print("B es mayor")
else:
    print("C es mayor")
